src/seedu/addressbook/AddressBook.java

```diff
@@ -1148,6 +1148,7 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
+     * TODO: Fix this function to avoid manipulating the input string
```

Comment:
while it is a good process to commit small changes, like i mentioned in the tutorial, such minor changes may not warrant commits.